README.md

```diff
@@ -75,27 +75,66 @@ Activate the virtual environment in Python
 
 `deactivate` to exit it from venv
 
-## `pip install -r requirement.txt`
+## `pip install -r requirements.txt`
 
 It will looks for the package in PyPI, resolves its dependencies, and installs everything in your current Python environment to ensure that requests will work.
 
 ## `pip freeze > requirements.txt`
 
-It records an environment's current package list into requirements. txt
-
-## `pip install uvicorn`
-
-Runs `pip install uvicorn` if you do not have uvicorn already as it is required.
+It records an environment's current package list into requirements.txt. If any new library is installed, make sure to run this command to store library version.
 
 ## `uvicorn main:app --reload`
 
 Runs the code inside the backend folder in order to start up the backend server.
 
 Make sure to run this before front-end server `yarn start` to fully functionable.
 
+## `How to host backend server on deta space`
+
+- Create an Account on Deta Space at [Sign Up](https://deta.space/signup) or [Login](https://deta.space/login) if you had one already.
+
+- Install Space CLI
+
+    ```
+    Mac: curl -fsSL <https://get.deta.dev/space-cli.sh> | sh
+    Linux: curl -fsSL <https://deta.space/assets/space-cli.sh> | sh
+    Windows: iwr <https://deta.space/assets/space-cli.ps1> -useb | iex
+    ```
+
+- Authenticate
+
+    ```
+    cd backend/
+    space login
+    ```
+- Access Token
+    - log in to your Deta Space. Enter your search bar and click "Settings."
+    - Click "Generate Token" and copy the new token and paste it to your terminal.
+
+- Create a project on DetaSpace
+    - create a new project within terminal  
+        ```
+        space new
+        ```
+
+    - go to Speacefile inside backend and change the name to same name as what you enter when ask in terminal 
+
+    - push to create a new project on DetaSpace
+        ```
+        space push
+        ```
+- Update Environment Variable
+
+    - Navigate to the project you've recently created, then click on the three dots located within a circle at the bottom right corner of the project square. From there, select 'Settings' and then 'Configuration'
+
+    - fill the following in ALLOW_CORS to allow Cors
+        ```
+        https://stbobax.netlify.app,http://localhost:3000
+        ```
+
```

Comment:
Missing important environment variable from the frontend (netlify) to connect with the newly deployed backend.
Otherwise, the site is showing error without any data to be loaded

Reply:
Addressed in 5a536fedc1e8c3c70df805286afc4ba8e130640e